packages/credential-registry/schemas/ClaimedCredentials.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "ClaimedCredentials",
+  "description": "The [[ClaimedCredentials]] holds the list of claimed [[ClaimedCredential]] by the user",
+  "type": "object",
+  "properties": {
+    "claimed": {
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/CeramicDocId"
```

Comment:
@PaulLeCam, is there any way to specify in this parent schema that all these Ceramic StreamID links should themselves conform to a specific schema (the ClaimedCredential schema in this case)?

Reply:
Yes, using the `$comment` field as defined in [CIP-82](https://github.com/ceramicnetwork/CIP/blob/main/CIPs/CIP-82/CIP-82.md).
It's just a hint for tooling though, it doesn't provide any runtime guaranty.

Reply:
cool, still seems like it would be useful to add now, especially as we might decide to add runtime validation in the future

Reply:
Added CIP-82 references